elyra/metadata/manager.py

```diff
@@ -21,25 +21,27 @@
 from jsonschema import validate, ValidationError, draft7_format_checker
 from traitlets import Type
 from traitlets.config import LoggingConfigurable
-from typing import Optional, List
+from typing import List
 
 from .metadata import Metadata
 from .schema import SchemaManager
 from .storage import MetadataStore, FileMetadataStore
 
 
 class MetadataManager(LoggingConfigurable):
+    """Manages metadata instances"""
 
     # System-owned namespaces
     NAMESPACE_RUNTIMES = "runtimes"
     NAMESPACE_CODE_SNIPPETS = "code-snippets"
     NAMESPACE_RUNTIME_IMAGES = "runtime-images"
 
-    metadata_class = Type(Metadata, config=True,
-                          help="""The metadata class.  This is configurable to allow subclassing of
-                          the MetadataManager for customized behavior.""")
+    metadata_store_class = Type(default_value=FileMetadataStore, config=True,
+                                klass=MetadataStore,
+                                help="""The metadata store class.  This is configurable to allow subclassing of
+                                the MetadataStore for customized behavior.""")
 
-    def __init__(self, namespace: str, store: Optional[MetadataStore] = None, **kwargs):
+    def __init__(self, namespace: str, **kwargs):
         """
         Generic object to read Notebook related metadata
         :param namespace: the partition where it is stored, this might have
```

Comment:
the `:param store:` (line 49) is no longer applicable/required

Reply:
Damn - great catch.  Thank you.

Reply:
I found a few things incorrect about that area and have submitted the change - thanks for your review.